pandas/io/formats/css.py

```diff
@@ -1,24 +1,39 @@
-"""Utilities for interpreting CSS from Stylers for formatting non-HTML outputs
+"""
+Utilities for interpreting CSS from Stylers for formatting non-HTML outputs.
 """
 
 import re
 import warnings
 
 
 class CSSWarning(UserWarning):
-    """This CSS syntax cannot currently be parsed"""
+    """
+    This CSS syntax cannot currently be parsed.
+    """
 
     pass
 
 
 def _side_expander(prop_fmt: str):
-    def expand(self, prop, value):
+    """
+    Parameters
+    ----------
+    prop_fmt : str
+    """
+
+    def expand(self, prop, value: str):
+        """
+        Parameters
+        ----------
+        prop
+        value : str
+        """
         tokens = value.split()
         try:
             mapping = self.SIDE_SHORTHANDS[len(tokens)]
         except KeyError:
             warnings.warn(
-                f'Could not expand "{prop}: {value}"', CSSWarning,
+                f"Could not expand '{prop}: {value}'", CSSWarning,
```

Comment:
Shouldn't need to change this; the intention was for double quotes within the error message